mqtt.py

```diff
@@ -51,6 +56,20 @@ def username(self):
   def password(self):
     return self._config['password'] if 'password' in self._config else None
 
+  @property
+  def ca_cert(self):
+    return self._config['ca_cert'] if 'ca_cert' in self._config else None
+
+  @property
+  def ca_verify(self):
```

Comment:
Can this be simplified ? For example

```
if 'ca_verify' in self._config:
  return self._config['ca_verify']
else
  return True
```

Reply:
I went back and forth.  I forced true/false to filter invalid user input in the configuration file.  Maybe it should care?

Remove it?  Keep it, but add comment to clarify?

Reply:
Comment would be good :D

Reply:
Comment added